newIDE/app/src/Profile/SubscriptionChecker.js

```diff
@@ -128,9 +127,8 @@ export class SubscriptionCheckDialog extends React.Component<
             <Star style={styles.icon} />
             <Text style={styles.iconText}>
               <Trans>
-                Having a subscription allows you to use the one-click export for
-                Android, Windows, macOS and Linux, launch live previews over
-                wifi, disable the GDevelop splashscreen during loading and more!
+                Having a subscription allows you to use the one-click export and
```

Comment:
One click export isn't locked by a plan,  there is 2 free export, just the user need an account with a free plan and is limited to 2 per day.